plugins/woocommerce/src/Admin/API/LaunchYourStore.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * REST API Launch Your Store Controller
+ *
+ * Handles requests to /launch-your-store/*
+ */
+
+namespace Automattic\WooCommerce\Admin\API;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Launch Your Store controller.
+ *
+ * @internal
+ */
+class LaunchYourStore {
+
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc-admin';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'launch-your-store';
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/dismiss-coming-soon-banner',
+			array(
+				array(
+					'methods'             => 'POST',
+					'callback'            => array( $this, 'dismiss_coming_soon_banner' ),
+					'permission_callback' => array( $this, 'must_be_shop_manager_or_admin' ),
+				),
+			)
+		);
+	}
+
+	public function must_be_shop_manager_or_admin() {
+		if ( ! current_user_can( 'shop_manager' ) && ! current_user_can( 'administrator' ) ) {
+			return false;
+		}
+		return true;
+	}
+
+	public function dismiss_coming_soon_banner() {
+		update_option( 'woocommerce_coming_soon_banner_dismissed', 'yes' );
```

Comment:
We should probably save this in the user meta, to account for multiple admins using the site 🤔

Reply:
@adrianduffell Good call! Updated it in [Use user meta to save woocommerce_coming_soon_banner_dismissed](https://github.com/woocommerce/woocommerce/pull/46149/commits/f4888a0e34679da391930442417bb2d78e4a02d6)

Reply:
Good point, @adrianduffell! On a related note, do you think we should change the status tour dismissal (https://github.com/woocommerce/woocommerce/pull/46203) to also be applied to user_meta too?

Reply:
@ilyasfoo @moon0326  Actually yeah -- it would be great to have user_meta applied on the tour dismissal too 👍